### PR TITLE
Add testnet-citus deployment to testnet-eu cluster

### PR DESCRIPTION
**Pull request by xin-hedera**
**Description**:

This PR adds testnet-citus deployment to testnet-eu cluster

- Add configs for testnet-citus to testnet-eu cluster
- Temporarily disable testnet-citus importer
- Temporarily disable testnet-citus ingress for grpc, rest, restjava, and web3

**Related issue(s)**:

Relates to #8188 

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

File: clusters/testnet-eu/testnet-citus/helmrelease.yaml [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+    global:
+      hostname: "*.mirrornode.hedera.com"
+    grpc:
+      ingress:
+        enabled: false
+    importer:
+      env:
+        HEDERA_MIRROR_IMPORTER_DOWNLOADER_RECORD_ENABLED: "false"
+        HEDERA_MIRROR_IMPORTER_NETWORK: "TESTNET"
+        HEDERA_MIRROR_IMPORTER_PARSER_RECORD_HISTORICALBALANCE_ENABLED: "false"
```
Comment (xin-hedera) [inline clusters/testnet-eu/testnet-citus/helmrelease.yaml:44]: effectively disable ingestion, `downloader.balance.enabled` defaults to `false`

**Review by steven-sheehy — COMMENTED** [review]
missing common chart changes and flux-system changes

File: clusters/testnet-eu/common/sealed-secret.yaml [hunk 2]
```diff
@@ -7,7 +7,7 @@ metadata:
...
...cQt6l3bPhZY+2CtLNK0vWR0/fmfW4ihcks3AaE9j/mOetxoaubo3dAzKwrlL/OumNqckeSGmzb47laboEaPtnnBsm/52yJGJwiaN7OV5eIBLmwh8OmQsEorsnVwLDZm9oln0dKLHN2KqyXLeEdn7u3wiM5VK9WFMPG0fVWbvTcEC+EHD6OEZrfEBOAo8wEstePCGPasqZb9NVnzVj+QTiUNwwqQaw7SB2oBfqZRsygJlqUGfcAVqBYN8fvJW4oowjqevUfewsxKtCd4MBboB1WCwmLKa0V7qp7rTpfptMG8QkslTPcVSFj++zOBkdsSOvJpmcB3pnT7g7C/XwQSuIDGyfytpfRIqQatAp40j8VLiFJFMUFBcmgHhNUerU5y8uRd3Em+LJwozKql6DrwhIBc7ENaT9+SPhSaZ+ivb+M/4qf2iSnNwlVLv4B5NXLFz2IXwkWBpv9KXTJ8K/zVtc6P3WIHC3EPtOWU767ejv+/sVNOF3kBcfgpFkfqW74V+CekMSlltjUCpb73l6dYLiJV3B9X/enFxO5eCkM+14ARWVW9Lmj0fYSJkZ4BKP2Fulc1L6olg/oHNgTacdm5j8JpC8F0LKdIPb0JGYDhmC0LqPRBv2GUDNsYeLLcjEOwf33e9vALqTNgiy/2GshwWqWnGNvUkb+fPFmwuZRy0IxRh0mYi4bK10ZfHL0cVh1hcpC3OeXsxHptbcSZz9+JCC5hTbqXvdMSRBM1vRnVJkLFTes3aoAoL572Ij55+p9v+HxhRrgwLRWoteZ//ZmuzI5nkZPxz0r+dVy6GeDuWL6VAQss9JohdrdsSROf0QSlwS3S4k/faUjqweaNu54/7ACA7WRa/g/D5D04h1jTfG1mwCNEB+hI4MI2X2rLZjfYMQuqojDf83BB/txkybOK1noivpVPbtaNcZgYZZVGcf/6Rv/UmIhypAR2h5IZeiqaP72VX8D3hMZ3jNhgsKH/0i8gt98sGp/EwcV47l7N/gLQY5CYFk+JqmG63DPE7w4bTQazMWka1RRt4e6YqY8jv3fZpBIbtsGxIFca7nYx34UPyKmsJxwcdkombt/UxbRMvE3l+DbLyvgmsHktFxnep0blXpmn/5pp9s/Vtc88B5HJLi0KWDtW57HxZw93QQJYMMxaiqkIyNaYloYIN4qiwBXZPH1QB6qCt55cXDgNkIl6cbA==
```
Comment (xin-hedera) [inline clusters/testnet-eu/common/sealed-secret.yaml:10]: add zfs init service account

**Review by steven-sheehy — CHANGES_REQUESTED** [review]

File: clusters/testnet-eu/testnet-citus/helmrelease.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (xin-hedera) [reply clusters/testnet-eu/testnet-citus/helmrelease.yaml:44]: I see, will make the change

File: clusters/testnet-eu/testnet-citus/helmrelease.yaml [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+    spec:
+      chart: hedera-mirror
+      sourceRef:
+        kind: HelmRepository
+        name: hedera-mirror-node
+      valuesFiles:
+        - values.yaml
+        - values-prod.yaml
+        - values-v2.yaml
+      version: '0.104.0-rc4'
```
[the remark this answers is not in the document]
Comment (xin-hedera) [reply clusters/testnet-eu/testnet-citus/helmrelease.yaml:17]: or we can just apply the migration v1.97.0 on a testnet db clone.

anyways there isn't much data in testnet, so probably just wait till we GA and deploy 0.104.0 to testnet

**Review by steven-sheehy — APPROVED** [review]
LGTM

File: clusters/testnet-eu/common/helmrelease.yaml [hunk 4]
```diff
@@ -49,8 +49,13 @@ spec:
...
+    stackgres:
+      enabled: true
     traefik:
       service:
         annotations:
           cloud.google.com/neg: '{ "exposed_ports":{ "443":{"name": "mirrornode-testnet-eu"} } }'
-
+    zfs:
+      enabled: true
+      init:
```
Comment (jnels124) [inline clusters/testnet-eu/common/helmrelease.yaml:60]: we should probably configure the default disk size to be 1000GI and remove the shard override configuration.

Comment (jnels124) [reply clusters/testnet-eu/common/helmrelease.yaml:60]: If we go the shard override approach, the correct syntax is below. The staging config is not correct currently.
```
      worker:
        overrides:
          - index: 1
            pods:
              persistentVolume:
                size: 1000Gi
          - index: 2
            pods:
              persistentVolume:
                size: 1000Gi
```

Comment (xin-hedera) [reply clusters/testnet-eu/common/helmrelease.yaml:60]: switched to set `zfs.worker.initialDiskSize` in the common charts config, and removed the worker override from hedera-mirrorr charts config

**Review by jnels124 — APPROVED** [review]
LGTM

**Pull request merged**